integrations/monetate/lib/index.js

```diff
@@ -46,6 +46,14 @@ Monetate.prototype.initialize = function() {
     };
   }
   window.monetateQ = window.monetateQ || [];
+
+  // Get monetateId from mt.v cookie.
+  var monetateIdCookie = document.cookie.match('(^|;) ?mt.v=([^;]*)(;|$)');
```

Comment:
I'm reading this regex as "pull anything between two semicolons, newlines included"

The below returned as a match when I validated it:
```
;mt.v=5.1879957417.1544197401555;
; mt.v=5.1879957417.1544197401555 console.log(process.env);
```

Adding in a `\w` requires the trailing semicolon to be adjacent to a char, but still allows for:
```
;mt.v=5.1879957417.154419740155 console.log(process.env) var a=0;
```

We'd want this expression to capture only the token, or `mt.v=<token>;`, correct?